### PR TITLE
Authentication Added

### PR DESCRIPTION
**Pull request by mo-kalaleeb**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 yes-soft-de
+Copyright (c) 2019 MickSawy3r
```
Comment (gh-assan) [inline LICENSE:3]: yes soft

File: java/com/sixbits/yessoft/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
+package com.sixbits.yessoft;
```
Comment (gh-assan) [inline java/com/sixbits/yessoft/MainActivity.java]: please use de.yessoft.android 
as root package name, for all classes

File: java/com/sixbits/yessoft/ui/NewsFeedFragment/NewsRecyclerView.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    public void onBindViewHolder(@NonNull NewsTextVH holder, int position) {
+
+    }
+
+    @Override
+    public int getItemCount() {
+        return 0;
+    }
+
+
```
Comment (gh-assan) [inline java/com/sixbits/yessoft/ui/NewsFeedFragment/NewsRecyclerView.java]: please do not keep this code, if you want you can find them in git history

**Comment by gh-assan** [comment]
good, thank you

**Comment by gh-assan** [comment]
in general use descriptive branch name and commit messages

**Review by Kenan-Hussein — APPROVED** [review]
good work, thank you

**Pull request merged**